src/main/java/cpp/projects/ProjectList.java

```diff
@@ -40,6 +41,7 @@ public void addProject(String title) {
      * @param title Name of the project
      */
     public void deleteProject(String title) {
+        assert (title != null && title != "") : "Cannot delete the project";
```

Comment:
I am not sure if the string variable can use `!=` in the `assert` statement. Should it be `title.equals("")` in this statement?